test/glific/interactive_templates_test.exs

```diff
@@ -318,4 +366,25 @@ defmodule Glific.InteractiveTemplatesTest do
     assert Enum.any?(translations["2"]["options"], fn option -> option["title"] == "परीक्षण 1" end)
     assert Enum.any?(translations["2"]["options"], fn option -> option["title"] == "परीक्षण 2" end)
   end
+
+  test "export the interactive template",
```

Comment:
Can we have tests for interactive_message_type "list" and "location_request_message"